server/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM debian:latest
+RUN DEBIAN_FRONTEND=noninteractive apt-get update
+RUN DEBIAN_FRONTEND=noninteractive apt-get install -yq tmux git python-dev \
+          python-mysqldb build-essential libmysqlclient-dev python-pip \
+          python-numpy python-sklearn netcat-traditional libmysqlclient-dev \
```

Comment:
The trick is to wait until the mysql server is ready, although Docker Compose employs
a "depends on utility" it doesn't really wait until the mysql server is actually ready,
so we iterate on a while 1 until we manage to open a connection.

This could be done in more...elegant way, but for now this is how I do it.